spring-integration-core/src/test/java/org/springframework/integration/message/HeaderMatcherTests.java

```diff
@@ -43,8 +42,9 @@
 import org.hamcrest.Matcher;
 import org.junit.Before;
 import org.junit.Test;
-import org.springframework.messaging.Message;
+
 import org.springframework.integration.support.MessageBuilder;
```

Comment:
I'd change this to the `org.springframework.messaging.support.MessageBuilder` rather than move the test stuff to the core...